Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -53,26 +149,87 @@ class ARExample: UIViewController {
         arView.stopTracking()
     }
 
-    private func makeStreetsScene() -> AGSScene {
+    /// Changes the scene to a newly selected scene.
+    ///
+    /// - Parameter sender: The bar button item tapped on.
+    @objc func changeScene(_ sender: UIBarButtonItem){
+        guard let label = currentSceneInfo?.label,
+            // Get the index of the scene currently shown in the sceneView.
+            let selectedIndex = sceneInfo.firstIndex(where: { $0.label == label }) else {
+                return
+        }
         
-        // create scene with the streets basemap
-        let scene = AGSScene(basemapType: .streets)
+        // Create the array of labels for the options table view controller.
+        let sceneLabels = sceneInfo.map { $0.label }
         
-        // create elevation surface
-        let elevationSource = AGSArcGISTiledElevationSource(url: URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!)
-        let surface = AGSSurface()
-        surface.elevationSources = [elevationSource]
-        surface.name = "baseSurface"
-        surface.isEnabled = true
-        surface.backgroundGrid.isVisible = false
-        scene.baseSurface = surface
+        // A view controller allowing the user to select the scene to show.
+        // Note: the `OptionsTableViewController` is copied from the "ArcGIS Runtime SDK for iOS Samples" code, found here:  https://github.com/Esri/arcgis-runtime-samples-ios
+        let controller = OptionsTableViewController(labels: sceneLabels, selectedIndex: selectedIndex) { [weak self] (newIndex) in
```

Comment:
`OptionsTableViewController` is handy in a navigation stack. But when presenting a list modally, I recommend using an action sheet instead. The currently selected action can be indicated by setting `isEnabled` to `false`.